src/cross/zif_dbbr_c_global.intf.abap

```diff
@@ -169,4 +169,5 @@ INTERFACE zif_dbbr_c_global
       scroll_to_bottom TYPE sy-ucomm VALUE 'PAGE_END',
     END OF c_function_codes ,
     c_for_all_entries_select_table TYPE string VALUE '<lt_for_all>' ##NO_TEXT.
-ENDINTERFACE.
+
+    endinterface.
```

Comment:
Execute pretty printer

Reply:
I executed pretty printer also in class zcl_dbbr_virtual_elem_handler. Was also missing